pkg/cmd/kafka/topic/list/list.go

```diff
@@ -61,9 +67,9 @@ func NewListTopicCommand(f *factory.Factory) *cobra.Command {
 		Long:    opts.localizer.MustLocalize("kafka.topic.list.cmd.longDescription"),
 		Example: opts.localizer.MustLocalize("kafka.topic.list.cmd.example"),
 		Args:    cobra.NoArgs,
-		RunE: func(cmd *cobra.Command, _ []string) error {
+		RunE: func(cmd *cobra.Command, _ []string) (err error) {
```

Comment:
Should we be using named return types, personally i dont think they are great but its personal preference for you